### PR TITLE
Add the possibility to deploy via script.

### PR DESCRIPTION
**Pull request by kud**

### CODE REVIEW

**Comment by kud** [comment]
@cirotix Ready for review. Wasn't able to make a loop for the moment. :)

**Comment by kud** [comment]
@cirotix Do not forget to add ssh key on the server for Yann.

File: bin/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env zsh
+
+echo "❯ Deploying…"
+for i in web1 web2;
```
Comment (cirotix) [inline bin/deploy.sh]: no need for `;` in a shell script

**Comment by kud** [comment]
Fixed.

**Comment by kud** [comment]
🎆

**Pull request merged**